uiowa/uiowa
# Feature: No upscaling for 'no crop' image styles and 'Original' image view mode

# Updates
Resolves #1974. 
* For default platform, I have removed upscaling from all the 'no crop' image styles. This should prevent them from appearing larger than the original image.
* Additionally, I have added the default image view mode to the collegiate text filters that allow images to be embedded. I also added some preprocessing code to rename 'Original' to 'Default,' similar to how it is done for SiteNow sites.

# To test
* `blt ds --site=cogscilang.grad.uiowa.edu`
* Edit the homepage: https://gradcogscilang.local.drupal.uiowa.edu/node/1/edit
* Insert a small image using the 'Small - no crop' option and observe whether it inserts the image as larger than it is.
* Click the 'edit media' button next to an existing image (you don't have to be able to see the image) and observe that the 'Original' option exists.
